gitman/models/source.py

```diff
@@ -336,16 +344,27 @@ def _invalid_repository(self):
         return exceptions.InvalidRepository(msg)
 
 
-def create_sym_link(source: str, target: str, *, force: bool):
-    log.info("Creating a symbolic link...")
+def create_sym_link(source: str, target: str, symbolic, *, force: bool):
```

Comment:
I think we should remove "sym" from the name if this will create two kinds of links:

```suggestion
def create_link(source: str, target: str, *, force: bool, symbolic:bool = True):
```